psydac/core/bsplines.py

```diff
@@ -476,14 +476,17 @@ def breakpoints( knots, degree ):
     degree : int
         Polynomial degree of B-splines.
 
+    tol: float
+        Maximum tolerance between the break points.
```

Comment:
I think it's best to be very explicit in this low-level module. So I suggest to add "If the distance between two knots is less than `tol`, we assume that they are repeated knots which correspond to the same break point."

Reply:
done